packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -139,7 +140,7 @@ function OrgNav({
   admin,
   hosts,
   currentProjectName = 'None',
-
+  isCloudManager = false,
```

Comment:
This will break the current cloud manager implementation, right?

Reply:
No, it shouldn't. OrgNav didn't need to know about whether or not it was on Cloud Manager until now.

Reply:
Only using this boolean to determine whether or not to display the See Product Tour link

Reply:
Sounds good – as long as that prop is added for CM when this version bump happens!